code-reviews/recipes/Go.md

```diff
@@ -14,6 +14,12 @@
 
 [golint](https://github.com/golang/lint) can be an effetive tool for finding many issues, but it errors on the side of false positives. It is best used by developers when working on code, not as part of an automated build process.
 
+## Useful extensions for VS Code and Visual Studio
+
+### vscode-go
+
+Using the Go extension for Visual Studio Code, you get language features like IntelliSense, code navigation, symbol search, bracket matching, snippets etc.
+
 ## Automatic Code Formatting
 
 ### gofmt
```

Comment:
While `gofmt` and `govet` are good tools to start within a build pipeline for code checking, they only check a few aspects of the code. Using an aggregate linter such as https://github.com/golangci/golangci-lint, can perform `gofmt` & `govet` plus a number of other tools from a single pipeline step with an integrated result. This is what a few large golang projects I participate in have moved to over to during the last year or so and it's improved our code checking and simplified our build scripts/pipelines.

Reply:
Another item about automatic linting & formatting is having guidance to add these checks in the VS Code IDE, since we are guiding users to install the go extension. The settings for the extension can be set to `golangci-lint` or other linters for inline warnings/errors within the IDE.

Reply:
Added how to use golangci-lint in VS Code. 
Also added, how to add golangci-lint as a build step in pipeline.